docs/src/drivers/gm.adoc

```diff
@@ -189,71 +181,80 @@ gm.<nr of card>.read
+set .counts-per-rev parameter correctly. When .index-mode
+is False and .index-enabled is true, .counts and .position
+are reset at channel-I pulse.
+l| .counts-per-rev     | (s32, R/V)          |
+Determine how many counts are between two index pulses. It
+is used only in round mode, so when both .index-enabled and
+.index-mode parameters are True. GM encoder process encoder signal
+in 4x mode, so for example in case of a 500 CPR encoder it should
+be set to 2000. This parameter can be easily measured by setting
+.index-enabled True and .index-mode False (so that .counts resets
+at channel-I pulse), than move axis by hand and see the maximum
+magnitude of .counts pin in halmeter.
+l| .index-invert       | (bit, R/W)          | When True, channel-I event (reset or round) occur on falling
   edge of channel-I signal, otherwise on rising edge.
-| .min-speed-estimate | (float, R/W)        | Determine the minimum measured velocity magnitude at which
-  .velocity will be set as nonzero. Setting this parameter too
-  low will cause it to take a long time for velocity to go to zero
-  after encoder pulses have stopped arriving.
-| .position-scale     | (float, R/W)        | Scale in counts per length unit. .position=.counts/.position-scale.
-  For example, if position-scale is 2000, then 1000 counts of the
-  encoder will produce a position of 0.5 units.
+l| .min-speed-estimate | (float, R/W)        |
+Determine the minimum measured velocity magnitude at which
+.velocity will be set as nonzero. Setting this parameter too
+low will cause it to take a long time for velocity to go to zero
+after encoder pulses have stopped arriving.
+l| .position-scale     | (float, R/W)        |
+Scale in counts per length unit. .position=.counts/.position-scale.
+For example, if position-scale is 2000, then 1000 counts of the
+encoder will produce a position of 0.5 units.
 |===
 
 .HAL example
 
 Setting encoder module of axis 0 to receive 500 CPR quadrature encoder signal and use reset to round position.
 
-----
-setp gm.0.encoder.0.counter-mode 0         # 0: quad, 1: stepDir
-setp gm.0.encoder.0.index-mode 1           # 0: reset pos at index, 1:round pos at index
-setp gm.0.encoder.0.counts-per-rev 2000      # GM process encoder in 4x mode, 4x500=2000
-setp gm.0.encoder.0.index-invert 0
-setp gm.0.encoder.0.min-speed-estimate 0.1 # in position unit/s
-setp gm.0.encoder.0.position-scale 20000   # 10 encoder rev cause the machine to
+[options="header"]
+|===
+| setp pin seetting                          | Comment
+l|setp gm.0.encoder.0.counter-mode 0         | 0: quad, 1: stepDir
+l|setp gm.0.encoder.0.index-mode 1           | 0: reset pos at index, 1:round pos at index
+l|setp gm.0.encoder.0.counts-per-rev 2000    | GM process encoder in 4x mode, 4x500=2000
+l|setp gm.0.encoder.0.index-invert 0         |
+l|setp gm.0.encoder.0.min-speed-estimate 0.1 | in position unit/s
+l|setp gm.0.encoder.0.position-scale 20000   | 10 encoder rev cause the machine to
```

Comment:
Formatting this as table looses the connection that it is a code example

Reply:
done